docs/source/ufs_utils.rst

```diff
@@ -665,3 +665,57 @@ Run script
 ----------
 
 To run, use the machine-dependent script under ./util/weight_gen
+
+***************************************************
+UFS_UTILS utilities
+***************************************************
+
+gdas_init
+=========
+
+Introduction
+------------
+
+The gdas_init utility is used to create coldstart initial conditions for global cycled experiments using the chgres_cube program.  It has two components: one that pulls the input data required by chgres_cube from HPSS, and one that runs chgres_cube. The utility is only supported on machines with access to HPSS:
```

Comment:
Since it makes forecast-only ICs too, suggest removing "cycled" from "global cycled experiments" or adding mention of forecast-only here as well.

Reply:
Done at 18ba48f.